react-toggle-switch/my-app/src/ToggleSwitch.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from 'react';
+
+export function ToggleSwitch() {
+  const [isOn, setIsOn] = useState(false);
+  let modifierClass = null;
+  function handleToggle() {
+    if (isOn) {
+      setIsOn(false);
+    } else {
+      setIsOn(true);
+    }
+  }
+  if (!isOn) {
```

Comment:
This seems backwards. Should the modifier be `is-on` when the switch is on? This sets it to `is-on` when it's off.

Oh. On line 21 if `isOn` is true you render 'OFF'. So it is backwards. :-) You should probably fix that.